### ISSUE TITLE
New Auto mode to loop through all stories in a single theme

### ISSUE THREAD

**Issue by rsargent** [CMU-CREATE-Lab/data-visualization-tools#68]
Three options:

- Loop through all waypoints from single story (already have)
- Loop through all waypoints from all stories in all themes (already have)
- Loop through all waypoints from all stories in a single theme (new feature request)

**Comment by arwright** [comment]
We particularly need this and need it to be stable before Davos.

**Comment by pdille** [comment]
Addressed in commit 7c696aace2044aad80837b8b2a5f86ed43ce6b4f

There are now 3 automode states you can be in.
  1. Loop through only a single story.
  2. Loop through all stories of a single theme
  3. Loop through all stories of all themes

The relevant settings in the config file to this are as follows:

```
  "enableAutoMode" : true,
  "autoModeCriteria" : {
    "cycleThroughStoriesAndThemes": true|false,
    "cycleThroughStories": true|false
  },
```
For state 1, you will want to set both 'cycleThroughStoriesAndThemes' and 'cycleThroughStories' to false. You then go into the story you want to have loop and press 'L' on the keyboard. This was (and still is) the default behavior.

For state 2, you will want to set 'cycleThroughStoriesAndThemes' to false and 'cycleThroughStories' to true. It was not possible to have automode be in the state before. Like usual, you go into the story you want and press 'L' on the keyboard. It will then cycle through the waypoints of this story and then all the other stories in the theme that this story is part of.

For state 3, you will want to set both 'cycleThroughStoriesAndThemes' and 'cycleThroughStories' to true. This was previously handled by the 'autoModeFullThemeCycle' config option. This option no longer exists now and is handled by the new setting above.
